docs/components/Chips/Chips.stories.mdx

```diff
@@ -144,3 +144,34 @@ colors for parity with Jobber's mobile platform, but we will not be implementing
 this is for the web components as of this first iteration of our web Chip. The
 mobile use-case is very specific to a filtering use case that we as of now do
 not anticipate will be required for the web.
+
+### Chips Composition with Standalone Chip (Web)
```

Comment:
```suggestion
### Chips composition with standalone Chip (Web)
```

Casing should follow our [content guidelines](https://atlantis.getjobber.com/?path=/docs/content-formatting--docs#capitalization)